website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -131,29 +131,16 @@ module "vpc" {
 module "eks" {
   # https://registry.terraform.io/modules/terraform-aws-modules/eks/aws/latest
   source  = "terraform-aws-modules/eks/aws"
-  version = "18.31.0"
+  version = "~> 19.0"
 
   cluster_name    = local.cluster_name
   cluster_version = "1.24"
 
   vpc_id     = module.vpc.vpc_id
   subnet_ids = module.vpc.private_subnets
 
-  # Required for Karpenter role below
-  enable_irsa = true
```

Comment:
This is the default so its redundant to show, but its provided here to be informative so lets keep it